asynchronix-util/src/lib.rs

```diff
@@ -1 +1,2 @@
+/// Observable states.
```

Comment:
Docstring to be deleted, otherwise cargo doc sees 2 docstrings for the same module (the one with `///` and the one with `//!`) and the module documentation reads "Observable states. Observable states."